draft-ietf-tls-tls13.md

```diff
@@ -2837,14 +2840,14 @@ As noted in {{zero-rtt-data}}, TLS provides only a limited
 inter-connection mechanism for replay protection for data sent by the
 client in the first flight.
 
-The "ticket_age" extension sent by the client SHOULD be used by
+The "masked_ticket_age" parameter in the client's "early_data" extension SHOULD be used by
 servers to limit the time over which the first flight might be
 replayed.  A server can store the time at which it sends a server
 configuration to a client, or encode the time in a ticket.  Then, each
-time it receives an early_data extension, it can check to see if the
+time it receives an "early_data" extension, it can unmask the value and check to see if the
 value used by the client matches its expectations.
 
-The "ticket_age" value provided by the client will be shorter than the
+The "masked_ticket_age" value provided by the client will be shorter than the
```

Comment:
Nit: Perhaps change 'The "masked_ticket_age" value' to 'The unmasked value' or similar? Otherwise this reads like you're talking about what the client sent rather than the true value. (Whereas the paragraph above says to unmask it.)